lesson_15/edmond_sabou/test_home1_les13.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+
+from lesson_15.edmond_sabou import home1_les13
+
+
+class TestPoligons(unittest.TestCase):
+
+    def test_if_side_is_not_none(self):
+        sides = home1_les13.Polygon = 1
```

Comment:
you need an instance of Polygon. You used the class (home1_les13.Polygon) an instance will be `home1_les13.Polygon(1,1,1,1)`

Some for the rest of the examples use instances, not the class.

Another thing are the asserts. If you want to test that an object is not none. test with the proper assert: `self.assertIsNotNone()`.

Reply:
@edisabou, can you please have another look at this PR and make the required changes

Reply:
instantiated classes. though could make `test_if_value_is_not_string` work. do you have any suggestions there please?